runtime/common/marshaling/index.ts

```diff
@@ -7,7 +7,11 @@ import {unmarshalDate} from './marshal_dates';
 import {unmarshalError} from './marshal_errors';
 import {unmarshalNumber} from './marshal_numbers';
 
+// JSON has no native way to represent `undefined`.
+const HACK_UNDEFINED_JSON_VALUE = '__UNDEFINED__';
```

Comment:
can we make this more specific to coda like `__CODA_UNDEFINED__`, just in case someone accidentally used the keyword.

Reply:
great idea, done.